phoenix-core/src/it/java/org/apache/phoenix/end2end/ConcurrentMutationsExtendedIT.java

```diff
@@ -132,7 +132,11 @@ public void testSynchronousDeletesAndUpsertValues() throws Exception {
         t2.start();
 
         doneSignal.await(60, TimeUnit.SECONDS);
+        IndexToolIT.runIndexTool(true, false, "", tableName, indexName, null,
+                0, IndexTool.IndexVerifyType.ONLY);
```

Comment:
Curious why we're verifying twice? IndexScrutiny is read-only.

Reply:
No, IndexScrutiny is not read only anymore since it triggers the read repair now.